devtools/pudl_to_postgres.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/bash
```

Comment:
We could create a README for dev tools explaining how to use pudl_to_postgres.sh and pyroscope.sh. 

The pudl_to_postgres could include:
- pudl_to_postgres installation explanation
- explanation of the lines that load to cloudsql

Reply:
devtools has probably got enough things in it that it should be rationalized somehow. I just had these scripts floating around locally and they seemed somewhat useful and I didn't want to lose them. Right now they're not generalized tools, just records of what worked for me in my environment. 

I tried to get the CloudSQL IAM authentication working and failed, so ended up just using the built in DB authentication, which I is probably not what we want longer term. Unless you can give it an explicit list of allowed IP addresses / ranges I think you need to connect through a CloudSQL proxy, or via the `gcloud` command line tools. For Mode/Hex/Tableau/Superset it probably works to provide allowed IP addresses, but for connecting from home it doesn't really.

It didn't seem like it was worth trying to generalize these tools and document all of what's going on here yet. What do you think?

Reply:
Yeah, probably not worth trying to generalize these tools yet. If we start using postges it will probably be built into the ETL and profiling could be built into CI.

Reply:
I think this is right. If we want to be able to load into postgres, we should create separate `to_sqlite()` and `to_postgresql()` methods for generating different SQLAlchemy schemas as required for the different DB backends, and allow a Postgres DB connection as well as an SQLite DB connection as an output endpoint.